README.md

```diff
@@ -319,6 +325,16 @@ To provide a custom checklist, use a file path for the first argument.
 
 The checklist format is described in detail [here](./docs/checklist-format.md).
 
+## Git Hooks
+
+The `rdm hooks` command installs git hooks that will automatically add change request numbers to your git commit messages, based on your branch name.
+
+> When using this feature, name your branches in the style of `${hyphen-separated-issue-numbers}-rest-of-branch-name`.
+>
+> Example: `12-13-fixing-issues`
+
+If you manually edit one of the provided hooks and want to revert your changes or RDM has been updated and you want to make sure the newest hooks are used, you can re-run this command at any time.
```

Comment:
Side-note: I noticed that we "install" our git hooks by copying the files over (via `copy_directory` util, which is then using `shutil.copy2`) - thus the reasoning for this note.

Any reason why we are not using symlinks? I think that would let us avoid upgrade issues, where the user has to manually re-run this install command in order to get upgraded hooks after pip upgrading.